utils/project-validator/config/config.yaml

```diff
@@ -8,3 +8,4 @@ services:
         resource: '../src'
         exclude:
             - '../src/ValueObject/*'
+            - '../src/Process/ParallelTaskRun.php'
```

Comment:
If this is value object, should be in `ValueObject` namespace. Thus the excluded case above